students/StefanLund/Lesson_2/fizz_buzz.py

```diff
@@ -0,0 +1,128 @@
+# python3
+
+# fizz_buzz.py
+# ---------------------------  START fizz_buzz_1(n)  ---------------------------
+def fizz_buzz_1(n):
+    """
+    returns Fizz if n is a multiple of 3
+    returns Buzz if n is a multiple of 5
+    returns FizzBuzz if n is a multiple of 3 and 5
+    returns n otherwise
+    """
+
+    if n % 3 == 0 and n % 5 == 0:
+        return("FizzBuzz")
+    elif n % 3 == 0:
+        return("Fizz")
+    elif n % 5 == 0:
+        return("Buzz")
+    else:
+        return(n)

```

Comment:
I'm not sure why there are so many versions of this but this here is the most common one except you would include the for loop over `range` function.